### DISCUSSION TITLE
redis配合mosdns的问题

### DISCUSSION THREAD

**Discussion by mengxinz** [IrineSistiana/mosdns#517] [Q&A]
redis需要设置哪些参数，有没有熟悉的发布一个教程

**Comment by Sakaizd** [comment]
docker run --name redis -d -p 6379:6379 redis

**Reply by mengxinz** [reply]
> Docker run --name redis -d -p 6379：6379 redis

我指的redis配置文件需要改哪些参数而不是容器命令

**Reply by Sakaizd** [reply]
直接用就行了 不用改
